packages/components/src/hooks/use-element-size.ts

```diff
@@ -0,0 +1,21 @@
+import type React from 'react';
+import { useEffect, useState } from 'react';
+import { getSizeFromDimensions, getElementSize, observeElementDimensions, observeWindowDimensions } from '../common';
+
+export const useElementSize = (element: React.RefObject<HTMLElement> | undefined, sizeAsHeight: boolean): number => {
```

Comment:
this is not expected that function with such name will observe window dimensions

Reply:
Discussed, will be executed later in stylable-components